examples/server_kube/README.md

```diff
@@ -1,3 +1,19 @@
 # Kubernetes ConfigMap example
 
-TODO(michal): Document this.
\ No newline at end of file
+- You need docker and kubernetes. [Docker Desktop](https://www.docker.com/products/docker-desktop), or you can use docker and [minikube](https://minikube.sigs.k8s.io/docs/start/)
+- initialize empty tmp mapping `mkdir -p /tmp/foobar` for kubernetes
+- run the server `go run .`
+
+you should see the following:
+```
+server2020/07/15 13:14:08 Now watching /tmp and /tmp/foobar
+server2020/07/15 13:14:08 etcd flag value watching initialized
```

Comment:
your example/doc is correct but the code is wrong
```
	logger.Printf("etcd flag value watching initialized")
```
should be "configmap ..." but let's not change that yet